src/lib/builders/accordion/create.ts

```diff
@@ -107,9 +114,9 @@ export const createAccordion = <T extends AccordionType = 'single'>(
 				};
 			};
 		},
-		action: (node: HTMLElement) => {
+		action: (node: HTMLElement): ActionReturn<unknown, TriggerEvents> => {
```

Comment:
Rather than pass unknown to ActionReturn, since no parameters are present, we should probably specify undefined.

Also, we could export a MeltActionReturn type, that accepts Parameters just the same, but for events it just accepts a list of event types, e.g. `MeltActionReturn<undefined, 'click' | 'keydown'>`, which automatically maps the event types to the appropriate event handlers. Wdyt?

Reply:
I think that's a great idea tbh!